index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
```

Comment:
You don't have a meta viewport in the head

Reply:
The same thing applies here 🤓

It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🧐 Kindly explain in your comments the reason for your requirements 😁

Also, please be sure to tell your requirements/suggestions in a way that it sounds like suggestions instead of orders 😉 The idea is to make it politely and friendly 😬